### PR TITLE
WFLY-12733 Proposal for EJB Timers: Forcing refresh timers in a datab…

### PR DESCRIPTION
**Pull request by chengfang**
…ase-data-store in a clustered environment.

JIRA:
https://issues.redhat.com/browse/WFLY-12733

Related Issue:
https://issues.redhat.com/browse/EAP7-1377

### CODE REVIEW

File: ejb/WFLY-12733-refresh-timer-programatically.adoc [hunk 1]
```diff
@@ -0,0 +1,120 @@
+= EJB Timers: Forcing refresh timers in a database-data-store in a clustered environment
+:author:            Cheng Fanbg
```
Comment (rhusar) [inline ejb/WFLY-12733-refresh-timer-programatically.adoc]: Who is `Fanbg`? :-)

Comment (chengfang) [reply ejb/WFLY-12733-refresh-timer-programatically.adoc]: typo fixed, thanks!

**Comment by chengfang** [comment]
I've just updated the proposal document to change to use EE interceptor (instead of WildFly custom interceptor) for configuring programmatic timer refresh. It should be a bit easier for users.

**Comment by chengfang** [comment]
@tadamski @fl4via I've updated the proposal a little bit, replacing the use of WildFly interceptor to EE interceptor.  Please review.  Thanks!

**Comment by chengfang** [comment]
A predefined interceptor will then be a public api with support promise, and there is not a good place (libary jar) to contain this class. On the other hand, if applications already have their interceptors, they can just add this simple logic there; otherwise, it should be simple for them to implement it.

**Pull request merged**